apps/files_sharing/src/utils/GeneratePassword.js

```diff
@@ -39,10 +40,12 @@ export default async function() {
 		try {
 			const request = await axios.get(config.passwordPolicy.api.generate)
 			if (request.data.ocs.data.password) {
+				showSuccess(t('files_sharing', 'Password was successfully created'))
 				return request.data.ocs.data.password
 			}
 		} catch (error) {
 			console.info('Error generating password from password_policy', error)
+			showError(t('files_sharing', 'Error generating password from password_policy: {error}', { error }))
```

Comment:
```suggestion
			showError(t('files_sharing', 'Error generating password from password_policy'))
```

you would very likely show a technical error to an end user